DataStructuresTest/HashTableTests.cs

```diff
@@ -8,6 +8,9 @@ namespace HashTableTests
     [TestClass]
     public class HashTableTests
     {
+        private const string _one = "one"; 
```

Comment:
1. Check out how constants should be called on the internet. 
2. It's rather AnyItem than _one.